public/js/controllers/index.js

```diff
@@ -1,10 +1,41 @@
+/*eslint-disable */
 angular.module('mean.system')
   .controller('IndexController', ['$scope', '$cookies', 'Global', '$http', '$location',
-    'socket', 'game', 'AvatarService', '$window',
+    'socket', 'game', 'AvatarService', 'userService',
     function IndexController($scope, $cookies, Global, $http, $location,
-      socket, game, AvatarService) {
-      $scope.global = Global;
+      socket, game, AvatarService, userService) {
+      $scope.global = Global.isAuthenticated();
       $scope.errorMsg = '';
+      $scope.showOptions = !$scope.global.authenticated;
+      const user =  $scope.global.user;
+
+      $scope.startGame = () => {
+        swal({
+          title: 'Start a new Game?',
+          text: 'You want to start the game now?',
+          type: 'success',
+          showCancelButton: true,
+          confirmButtonColor: '#DD6B55',
+          confirmButtonText: 'Yes',
+          cancelButtonText: 'No',
+          closeOnConfirm: true,
+          closeOnCancel: true
+        },
+        (isConfirm) => {
+          if (isConfirm) {
+            const data = {
+              gameOwnerId: user.id,
+              players: [user.id]
+            };
+            userService.startGame(data).then(({ data }) => {
+              Global.setCurrentGameId(data._id);
+              $location.path('/app');
+            });
+          } else {
+            swal('Cancelled', 'You are off! Shitty you!!!', 'error');
```

Comment:
uhm , can you use a proper message like game, cancelled.

And add (error) => {} callback where you check the status code and redirect the user to login page or show a server error message.